src/main.d

```diff
@@ -172,11 +183,18 @@ int main(string[] args)
 		if (cfg.getValueBool("resync")) safeRemove(syncListHashFile);
 	}
 	
+	if ((!exists(cfg.configDirName ~ "/business_shared_folders")) && (exists(businessSharedFoldersHashFile))) {
+		// if --resync safe remove business_shared_folders.hash
+		if (cfg.getValueBool("resync")) safeRemove(businessSharedFoldersHashFile);
+	}
+	
 	// Read config hashes if they exist
 	if (exists(buildNormalizedPath(cfg.configDirName ~ "/config"))) currentConfigHash = computeQuickXorHash(buildNormalizedPath(cfg.configDirName ~ "/config"));
 	if (exists(buildNormalizedPath(cfg.configDirName ~ "/sync_list"))) currentSyncListHash = computeQuickXorHash(buildNormalizedPath(cfg.configDirName ~ "/sync_list"));
+	if (exists(buildNormalizedPath(cfg.configDirName ~ "/business_shared_folders"))) currentBusinessSharedFoldersHash = computeQuickXorHash(buildNormalizedPath(cfg.configDirName ~ "/business_shared_folders"));
```

Comment:
That code `buildNormalizedPath(cfg.configDirName ~ "/business_shared_folders")` appears several times and can be moved to line 151 and only initialized once

Reply:
Hrmm ... will check .. forest / trees and the like!

Reply:
Once #961 is merged into 'master', that will update most of these, then will update the remaining from this PR

Reply:
https://github.com/abraunegg/onedrive/pull/473/commits/2304df10770de3e6401ddcf485105fdde54d5c7f fixes this